src/ImageSharp/Formats/Tiff/TiffDecoderCore.cs

```diff
@@ -50,9 +50,9 @@ public TiffDecoderCore(Configuration configuration, ITiffDecoderOptions options)
         }
 
         /// <summary>
-        /// Gets or sets the number of bits per component of the pixel format used to decode the image.
+        /// Gets or sets the bits per sample.
         /// </summary>
-        public TiffBitsPerSample BitsPerSample { get; set; }
+        public ushort[] BitsPerSample { get; set; }
```

Comment:
@brianpopow 👍  it looks easier and better

Reply:
I'm curious, why is this a public property (_I haven't reviewed the usage_)? I'm always cautious around exposing arrays.

Reply:
@JimBobSquarePants oh yes, arrays aren't safe, can make as `ReadOnlyCollection`

Reply:
We reuse the property for TiffFrameMetadata. Surely there’s limited scope for allowable bits per sample no?

Reply:
Thanks! Ok, then we're gonna have to sanitize that property or use some sort of struct that takes up to 3 values. 

I see [Aspose](https://apireference.aspose.com/imaging/net/aspose.imaging.imageoptions/tiffoptions/properties/bitspersample) use an array

Reply:
I have added back in now the `TiffBitsPerSample` enum instead of using the array directly: 67f7b782